include/bluetooth/gap.h

```diff
@@ -69,6 +69,30 @@ extern "C" {
 #define BT_GAP_INIT_CONN_INT_MIN                0x0018  /* 30 ms    */
 #define BT_GAP_INIT_CONN_INT_MAX                0x0028  /* 50 ms    */
 
+/* LE PHY types */
+#define BT_GAP_LE_PHY_1M                        0x01 /* LE 1M PHY */
+#define BT_GAP_LE_PHY_2M                        0x02 /* LE 2M PHY */
+#define BT_GAP_LE_PHY_LONG_RANGE                0x03 /* LE Coded PHY */
+
+/* Advertising PDU types */
+#define BT_GAP_ADV_TYPE_ADV_IND                 0x00
```

Comment:
Technically speaking we should use the GAP terminology here:

`BT_GAP_ADV_TYPE_CONN`
`BT_GAP_ADV_TYPE_NON_CONN`
...

But your proposal makes it easier to match to sniffer traces and the HCI spec, so not worse per-se.

Reply:
At least please add comments that reflect the GAP name:

`/* Scannable and connectable advertising */`
`/* Directed connectable advertising */`
`/* Non-connectable and scannable advertising */`
`/* Non-connectable and non-scannable advertising */`
...

Reply:
I needed to separate it from the HCI defines,  but we could call it BT_ADV_TYPE_<NAME> as well.
(Since the HCI ones are BT_LE_) and we might want to move away from LE in the names.

Reply:
Missing ADV_TYPE_EXTENDED